server/container/guice/pom.xml

```diff
@@ -73,6 +73,7 @@
         <module>protocols/webadmin-mailrepository</module>
         <module>protocols/webadmin-rabbitmq-mailqueue</module>
         <module>protocols/webadmin-swagger</module>
+        <module>protocols/webadmin-mail-over-web</module>
```

Comment:
I propose to not add yet-another module but add it to `protocols/webadmin-mailqueue`